ros2_test_cases_stats.py

```diff
@@ -41,6 +44,24 @@
         author {
           login
         },
+        timelineItems (first: 100, itemTypes: [CROSS_REFERENCED_EVENT]) {
```

Comment:
I assumed 100 would be plenty, especially since this is filtering on `CROSS_REFERENCED_EVENT`, so if I'm not mistaken, the issue has to be cross referenced more than 100 times before this becomes a problem.